src/screens/Login/Login.js

```diff
@@ -0,0 +1,53 @@
+import React, {Component} from 'react';
+import {Text, View, ScrollView, Image, TextInput} from 'react-native';
+import CustomButton from 'src/components/CustomButton';
+import styles from './styles';
+
+class Login extends Component {
+  state = { login: '', password: '' };
+
+  static navigationOptions = { header: null };
```

Comment:
You also may use `defaultNavigationOptions` to hide headers for all stack navigator  screens .

[createStackNavigator](https://reactnavigation.org/docs/en/stack-navigator.html#stacknavigatorconfig)